admin/src/app/login/login.component.scss

```diff
@@ -0,0 +1,168 @@
+.logo-avatar {
+    width: 40px;
+    height: 39px;
+    opacity: 1;
+    border-radius: 5px;
+}
+
+.logo-image {
+    width: 40px;
+    height: 39px;
+    border-radius: 3px;
+
+}
+
+.header-text {
+    width: 133px;
+    height: 35px;
+    color: #FFFFFF;
+    opacity: 1;
+    font: W4 32px/54px Hiragino Maru Gothic ProN;
+    letter-spacing: 0;
+    text-align: center;
+}
+
+.header-space-div {
+  width: 5px;
+}
+
+.header-card-space-div {
+  height: 20px;
+}
+
+.new-account {
+  font-size: 10px;
+}
+
+.login-page {
+  padding: 0;
+  margin: 0;
+}
+
+.image-header {
+  // text-align: center;
+  // vertical-align: middle;
+  position: relative;
+  display: table; 
+
+  // position: relative;
+
+  // transform: translate(-50%, -50%);
+
+}
+
+.image-header p {
```

Comment:
write css according to sass syntax https://sass-lang.com/documentation/syntax/parsing

Reply:
done